mysys/perf_counters.cc

```diff
@@ -0,0 +1,188 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+#include "perf_counters.h"
+#include <cassert>
+#include <memory>
+
+#if defined(FB_DYNO) && FB_DYNO
+#include <dyno_counters.h>
+#endif
+
+#include <ctime>
+
+#define diff_timespec(TS1, TS2) \
+  ((TS1.tv_sec - TS2.tv_sec) * 1000000000ULL + TS1.tv_nsec - TS2.tv_nsec)
+
+////////////////////////////////////////////////////////////////////////////////
+namespace utils {
+
+template <class T>
```

Comment:
Seems like a not very useful alias. Fully-qualified `utils::unique_ptr` is not used anywhere. Moreover, there is no alias for std::shared_ptr which is used alongside with this unique_ptr.
Should be removed in my opinion.

Reply:
This is because of different template params:
```
template< class T > class shared_ptr;
template< class T, class Deleter = std::default_delete<T> > class unique_ptr;
```

Reply:
Oh, I see now